Framework/Beamline/src/ComponentInfo.cpp

```diff
@@ -336,8 +341,10 @@ ComponentInfo::Range
 ComponentInfo::componentRangeInSubtree(const size_t index) const {
   const auto rangesIndex = compOffsetIndex(index);
   const auto range = (*m_componentRanges)[rangesIndex];
-  return {m_assemblySortedComponentIndices->begin() + range.first,
-          m_assemblySortedComponentIndices->begin() + range.second};
+  ComponentInfo::Range ret = {
+      m_assemblySortedComponentIndices->begin() + range.first,
+      m_assemblySortedComponentIndices->begin() + range.second};
+  return ret;
```

Comment:
Why this change? I cannot spot a difference?

Reply:
I  think this is a relic from some previous debugging I was doing, I'll change it back.